packages/@lwc/signals/README.md

```diff
@@ -0,0 +1,75 @@
+# @lwc/signals
+
+This is an experimental package containing the interface expected for signals.
+
+A key point to note is that when a signal is both bound to an LWC class member variable and used on a template,
+the LWC engine will attempt to subscribe a callback to rerender the template.
```

Comment:
I wrote a quick `readme` based on our internal docs and the RFC that caridy wrote. Open to changing the wording on any of this!

Reply:
As mentioned above, I think for now this should probably just be in `@lwc/engine-core`. We can decide later if we want to expose a brand-new package out of this.

Reply:
@nolanlawson let's chat offline about this. Having a separate package will help facilitate the related work that is coevolving.